x/auth/simulation/genesis.go

```diff
@@ -91,36 +91,15 @@ func RandomizedGenState(simState *module.SimulationState) {
 }
 
 // RandomGenesisAccounts returns randomly generated genesis accounts
-func RandomGenesisAccounts(simState *module.SimulationState) (genesisAccs exported.GenesisAccounts) {
-	for i, acc := range simState.Accounts {
+func RandomGenesisAccounts(simState *module.SimulationState) (genesisAccs []exported.GenesisAccount) {
```

Comment:
Thoughts on the best place for this function?

Reply:
Great question. Each module is passed a `SimulationState` param to their respective `RandomizedGenState` function. So instead, perhaps we can have `SimulationState` now also include a customizable `AccountGenerator` function. Then, the `RandomizedGenState` in `x/auth` will utilize this generator instead of the current `RandomGenesisAccounts`.

Also, we should verify if we even need `Accounts` in the `SimulationState` (cc @fedekunze).